packages/amplify-schema-validator/.gitignore

```diff
@@ -0,0 +1,7 @@
+# dependencies
```

Comment:
Probably don't need/want a new gitignore in this package, can leverage the root one.